### PR TITLE
Fix imx6ull building and image creation

### PR DESCRIPTION
**Pull request by MaciejPurski**

### CODE REVIEW

**Review by pawelpisarczyk — COMMENTED** [review]
Maciej, I'm not sure if this is a good idea to disable OpenVPN compilation. You can do this by defining variable in specific project, but I think that in generic project these ports should be included in the image.

**Comment by MaciejPurski** [comment]
> Maciej, I'm not sure if this is a good idea to disable OpenVPN compilation. You can do this by defining variable in specific project, but I think that in generic project these ports should be included in the image.

I'm ok with that. So I'll get rid of the first commit.
For now, I don't understand, why the CI build fails on zynq7000.

File: build.project [hunk 1]
```diff
@@ -69,6 +69,7 @@ b_build() {
 #
 b_image() {
 	b_image_target
+	declare -F b_image_project && b_image_project
```
Comment (nalajcie) [inline build.project:72]: this returns 1 which makes build.sh exit with return code 1

Comment (MaciejPurski) [reply build.project:72]: Ok. In the next version I'l try something like this:
```suggestion
	declare -F b_image_project && b_image_project
	return 0
```

**Pull request merged**